.github/workflows/jetbrains-updates-template.yml

```diff
@@ -59,12 +59,24 @@ jobs:
             This PR updates the ${{ inputs.productName }} IDE image to the latest release version.
 
             ## How to test
-            Go to the preferences and select ${{ inputs.productName }} as desktop IDE. Start a workspace and verify that the new IDE version works.
+            1. Open the preview environment generated for this branch
+            2. Choose ${{ inputs.productName }} as your preferred editor
+            3. Start a workspace using this repository: `https://github.com/gitpod-io/spring-petclinic`
+            4. Verify that the workspace starts successfully
+            5. Verify that the IDE opens successfully
+            6. Verify that the version of the IDE corresponds to the one being updated in this PR
+
+            The following resources should help, in case something goes wrong (e.g. workspaces don't start):
+
+            - https://www.gitpod.io/docs/troubleshooting#gitpod-logs-in-jetbrains-gateway
+            - https://docs.google.com/document/d/1K9PSB0G6NwX2Ns_SX_HEgMYTKYsgMJMY2wbh0p6t3lQ
 
             ## Release Notes
             ```release-note
             Update ${{ inputs.productName }} IDE image to version ${{ steps.latest-release.outputs.version }}.
             ```
+
+            _This PR was created automatically via [this](https://github.com/gitpod-io/gitpod/blob/main/.github/workflows/jetbrains-updates-template.yml) GitHub Actions workflow_
```

Comment:
It's actually this one JB Releases https://github.com/gitpod-io/gitpod/blob/main/.github/workflows/jetbrains-updates.yml

Reply:
the link goes to the template where text can be updated

Reply:
I will double-check what to link to

Reply:
@mustard-mh I would really like to link to the template, so I've changed the words to be correct. What do you think?

Reply:
Up to you 👍